extensions/spark/kyuubi-spark-authz/pom.xml

```diff
@@ -336,6 +336,23 @@
     </build>
 
     <profiles>
+        <!--
+          Add hudi profile here to avoid import Hudi when enable scala-2.13.
+          Can remove this profile after hudi support Scala 2.13.
+          https://issues.apache.org/jira/browse/HUDI-6296
+          -->
+        <profile>
+            <id>hudi</id>
```

Comment:
Let's name the id more specifically, how about spark-authz-hudi-test?

Reply:
Emm ... I suppose the lineage module may reuse this profile in the future

Reply:
I'm sorry for not following a previous discussion and raising a new one may cause back-and-forth. IMHO, if we use -Phudi here, developers might get confused about what they actually enable.

Reply:
What do you think about enriching the comments with

> Currently, Hudi does not support Scala 2.13 (tracked via HUDI-6296), and Hudi is used in authz module for testing. The profile is used for conditional importing Hudi dependencies.

Reply:
> Let's name the id more specifically, how about spark-authz-hudi-test?

Done, it's more clear since this profile just for hudi test.